overlord/auth/auth.go

```diff
@@ -117,6 +131,88 @@ func User(st *state.State, id int) (*UserState, error) {
 	return nil, fmt.Errorf("invalid user")
 }
 
+// SetDeviceIdentity sets the device's identity as used for communication with the store.
+// XXX: Should eventually require the serial assertion to be the
+// assertions DB rather than taking it directly, but that requires
+// significant work.
+// XXX: Doesn't check that the serial assertion is trustworthy, just
```

Comment:
Sorry, small correction: _something_ needs to do that. This is probably not the place to do it. We probably don't even need the assertion here.